base_tier_validation/models/tier_validation.py

```diff
@@ -372,3 +377,44 @@ def _update_counter(self):
     def unlink(self):
         self.mapped("review_ids").unlink()
         return super().unlink()
+
+    @api.model
+    def fields_view_get(
+        self, view_id=None, view_type="form", toolbar=False, submenu=False
+    ):
+        res = super().fields_view_get(
+            view_id=view_id, view_type=view_type, toolbar=toolbar, submenu=submenu
+        )
+        if view_type == "form" and not self._tier_validation_manual_config:
+            doc = etree.XML(res["arch"])
+            for node in doc.xpath(self._tier_validation_buttons_xpath):
+                # By default, after the last button of the header
+                str_element = self.env["ir.qweb"]._render(
+                    "base_tier_validation.tier_validation_buttons"
+                )
+                new_node = etree.fromstring(str_element)
+                for new_element in new_node:
+                    node.addnext(new_element)
+            for node in doc.xpath("/form/sheet"):
```

Comment:
As we can set param _tier_validation_buttons_xpath, as start point of validation buttons,
Should we have param `_tier_validation_label_xpath` also? In case we want to add it to form without sheet.
WDYT?

Reply:
Otherwise, function tested and code reviewed, LGTM :+1:

Reply:
Well, I don't think so, as it has a specific behavior, showing a div before the sheet and showing a specific widget after. WDYT?

Reply:
I am ok, if it is intention. User still have option to do it manually.